client-side/src/components/settings/editUserProfile/ProfileImageEdit.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import GenericButton from '../../../stories/Button/GenericButton';
+
+const ProfileImageEditButton = () => {
+  const [file, setFile] = useState(null);
+  const [preview, setPreview] = useState(null);
+  const userId = '6694d2295d41f7809588274c';
+  const [imageFile,setImageFile]= useState(null);
+  const url=process.env.REACT_APP_BASE_URL;
+
+  const handleFilePicture = (e) => {
+    if (e) {
+      console.log('at handleFilePicture the  file is ' ,e.target.files[0]);
+      setImageFile(e.target.files[0]);
+      const reader = new FileReader();
+      reader.onloadend = () => {
+        setPreview(reader.result);
+      };
+      if(e.target.files[0])
+      {
+         reader.readAsDataURL(e.target.files[0]);
+      }
+    }
+  };
+
+  const handleUploadPicture = async () => {
+    const formData = new FormData();
+    formData.append('profileImage', imageFile);
+    try {
+      const response = await axios.put(`http://localhost:5000/users/${userId}`, formData, {
```

Comment:
use ${url}
not http://localhost:5000
for exemple:`${baseUrl}/preferences/${preferenceId}`